packages/code-block/src/prism.ts

```diff
@@ -7876,8 +7876,8 @@ export { Prism };
       return "string" == typeof e
```

Comment:
It seems to be failing formatting check due to the changes here, were they done by prettier? IIRC this file should be ignored.

Reply:
Hello, prettier made changes at this file when I run format

Reply:
Weirdly, if I do format on my machine, it changes this file back to what it was before. It might be a Unix-Windows thing, not sure, adding it to a `.prettierignore` either way.